js/blocks-image-customization.js

```diff
@@ -0,0 +1,53 @@
+const {assign} = lodash;
+
+function largo_core_image_block_add_media_credit( element, blockType, attribute ){
+
+    if( blockType.name === 'core/image' ){
+
+        var attachment_id = attribute.id;
+        var media = new wp.api.models.Media( { id: attachment_id } );
+
+        var media_response = media.fetch().then( function( media ) {
+
+            // the original instance of the media caption
+            var attachment_caption = attribute.caption;
+
+            if( media.media_credit && media.media_credit._media_credit ){
+
+                // our default media credit, if it exists 
+                var media_credit = '<span class="largo-attachment-media-credit">' + media.media_credit._media_credit + '</span>';
```

Comment:
The existing markup for this is:

```
<p class="wp-media-credit">credit / organization</p>
```

The generator for that markup is https://github.com/INN/largo/blob/0.5-dev/lib/navis-media-credit/navis-media-credit.php#L242-L273